tasty-query/shared/src/main/scala/tastyquery/reader/classfiles/ClassfileParser.scala

```diff
@@ -140,6 +140,7 @@ private[reader] object ClassfileParser {
         val bytesArrArg = scalaSigAnnotation.values.head._2.asInstanceOf[AnnotationValue.Arr]
         val idxs = bytesArrArg.values.map(_.asInstanceOf[AnnotationValue.Const].valueIdx)
         pool.sigbytes(idxs)
+      case _: SimpleName => failNoAnnot()
```

Comment:
This was reported as an inexhaustive match. I figured failing seemed reasonable but I'm happy to change it if you think there's a better option @sjrd